src/base/emu-i386/simx86/trees.c

```diff
@@ -970,6 +971,7 @@ TNode *Move2Tree(IMeta *I0, CodeBuf *GenCodeBuf)
 	}
 	/* ->REPLACE the code of the node found with the latest
 	   compiled version */
+	e_unmarkpage(nG->seqbase, nG->seqlen);
```

Comment:
Is this needed?
It should be unmarked at that point
because the one you added to
TraverseAndClean() should be enough?

Reply:
yes it's needed though only for alive nodes (fixed now, check missing in initial patch).

TraverseAndClean only clears nodes that are aged, this one replaces nodes that are not aged.